css/css-flexbox/reference/percentage-descendant-of-anonymous-flex-item-ref.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Flexbox Test: percentage heights in descendants of anonymous flex items</title>
+<link href="resources/flexbox.css" rel="stylesheet">
```

Comment:
You don't need this in the reference file.

Reply:
Ups, good point.